src/Application.vala

```diff
@@ -1,8 +1,33 @@
+/*
+* Copyright (c) 2021 The Photostat Project
+*
+* This file is part of Photostat.
+*
+* Photostat is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+* Photostat is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+* You should have received a copy of the GNU General Public License
+* along with Photostat. If not, see <https://www.gnu.org/licenses/>.
+*
+* Authored by: Abdallah "Abdallah-Moh" Mohammad <abdullah_mam1@icloud.com>
+* Authored by: Rajdeep "Suzie97" Singha <singharajdeep97@gmail.com>
+*/
+
+namespace Photostat {
```

Comment:
Is creating the extra namespace really necessary? Can we move it under the `Photostat.Application` class?